pkg/executor/show.go

```diff
@@ -822,6 +832,7 @@ func (e *ShowExec) fetchShowIndex() error {
 			"YES",            // Index_visible
 			nil,              // Expression
 			"YES",            // Clustered
+			isGlobalIndex,    // Global_index
```

Comment:
So here you can just use "NO"
```suggestion
			"NO",    // Global_index
```

Reply:
Ah, sorry, I had not indented it correctly :(